packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -351,6 +355,9 @@ class ExpansionPanelList extends StatefulWidget {
   /// constructor.)
   final Object initialOpenPanelValue;
 
+  /// Padding is applied to entire panel when expanded
```

Comment:
Since the padding is applied to the body of the `ExpansionPanel` and not the entire panel, consider renaming it and perhaps describing how a developer might use this parameter.

Reply:
You are right, will be fixed. I will try to explain the default behavior + describe the parameter usage by giving examples.